src/traces/parcats/attributes.js

```diff
@@ -0,0 +1,182 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var extendFlat = require('../../lib/extend').extendFlat;
+var colorAttributes = require('../../components/colorscale/attributes');
+
+var scatterAttrs = require('../scatter/attributes');
+var scatterMarkerAttrs = scatterAttrs.marker;
+var colorbarAttrs = require('../../components/colorbar/attributes');
+
+var marker = extendFlat({
```

Comment:
This seems more like `line` to me, and that would also be consistent with `parcoords`.

Reply:
Sounds good. I think I started out calling it `marker` because I thought it might be nice to style the outline of the paths, and to call this `line`.  But I don't think that really makes sense anymore.  I'll change in to `line`.

Reply:
Done in https://github.com/jonmmease/plotly.js/pull/1/commits/7b751009fc9804272316f0bb539ed0386c0858bd